WebDriverAgentLib/Utilities/FBImageIOScaler.h

```diff
@@ -10,6 +10,8 @@
 #import <Foundation/Foundation.h>
 #import <CoreGraphics/CoreGraphics.h>
 
+@class UTType;
```

Comment:
Just a note:
This type has worked since iOS 14. We no longer support the version, but current WDA can work with iOS 13. So after this version, such a lower version completely does not work as incompatible code while can compile and install to lower iOS version such as iOS 12 and 13.